src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+(unreleased)
+++++++++++++++++++
+* vm: makes image create's '--source' argument mandatory, also improves the clarity of the help 
```

Comment:
I can drop the `help` stuff. About `BC`, syntax wise it is, but w/o the `--source`, the command will never work, so it is de facto mandatory. BC is meant to tell people they need to change their command, but they really have nothing to change here.